### PR TITLE
Initial prototype of a new homepage with streams

### PR DESCRIPTION
**Pull request by clone1018**
Implements ideas drafted in https://github.com/Glimesh/glimesh.tv/discussions/605

## Homepage
Mostly redesigned homepage, fully translated. Shows 6 random streams for an hour, will persist through deploys / reboots as well.
image

## Alpha Page
Moved content from the homepage to this page, added translatable text.
image

## Stream Settings Page
Reorganized content a bit, added some clarifying text, and a new homepage section at the bottom.
image

### CODE REVIEW

**Comment by clone1018** [comment]
Coverage is jumping back and forth for me, will increase it....

File: lib/glimesh_web/live/user_settings/components/channel_settings_live.html.leex [hunk 1]
```diff
@@ -167,6 +179,31 @@
...
 </div>
+
+<h2 class="mt-4"><%= gettext("Homepage") %></h2>
+<p><%= gettext("Glimesh is now showing your channels on the homepage! It's a completely random process, however you must opt-in and you must have completed the following qualifications first.") %></p>
+<div class="row mb-4">
+    <div class="col-sm-6">
+        <strong><%= gettext("Homepage Qualifications") %></strong>
+        <ul>
+            <li><%= gettext("Show on Homepage must be checked") %></li>
+            <li><%= gettext("Must have streamed for at least 10 hours") %></li>
```
Comment (mja00) [inline lib/glimesh_web/live/user_settings/components/channel_settings_live.html.leex:190]: Possibly to clarify any confusion, unless I am the one who is confusion.
```suggestion
            <li><%= gettext("Must have streamed for at least 10 hours total on Glimesh") %></li>
```

Comment (mja00) [reply lib/glimesh_web/live/user_settings/components/channel_settings_live.html.leex:190]: I would say the message on line 192 would clarify this one but some people might misunderstand. Especially in translations. :)

**Review by mja00 — APPROVED** [review]
Preliminary approval, I haven't pulled it down and tested but it looks fine from what I can see, minus my comment but it's subjective.

**Pull request merged**